packages/vite/src/node/plugins/resolve.ts

```diff
@@ -104,13 +108,19 @@ export function resolvePlugin(baseOptions: InternalResolveOptions): Plugin {
       const targetWeb = !ssr || ssrTarget === 'webworker'
 
       // this is passed by @rollup/plugin-commonjs
-      const isRequire =
+      const isRequire = !!(
         resolveOpts &&
         resolveOpts.custom &&
         resolveOpts.custom['node-resolve'] &&
         resolveOpts.custom['node-resolve'].isRequire
+      )
```

Comment:
Maybe we want to shorten this to `resolveOpts?.custom?.['node-resolve']?.isRequire ?? false` if this is possible

Reply:
Yeah, it does look better!